omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs2_0/EncounterSearchHandler2_0.java

```diff
@@ -58,7 +57,8 @@ public SearchConfig getSearchConfig() {
 	public PageableResult search(RequestContext context) throws ResponseException {
 		String patientUuid = context.getRequest().getParameter("patient");
 		String encounterTypeUuid = context.getRequest().getParameter("encounterType");
-		
+		String[] visitsUuid=context.getRequest().getParameter("visits").split(",");
```

Comment:
Better to use existing method:  `String[] visitUuids = context.getRequest().getParameterValues("visit");`